### PR TITLE
Feature/276 login screen ui

### PR DESCRIPTION
**Pull request by Rebal67**

### CODE REVIEW

File: src/containers/SSOLoginContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+                    </View>
+                </View>
+            }
+            footerHint={
+                <View style={styles.infoBox}>
+                    <TInfoBox
+                        align="left"
+                        icon="security"
+                        description="100% secure. Only your phone can authorize your app login."
+                        linkUrl="#"
```
Comment (theblockstalk) [inline src/containers/SSOLoginContainer.tsx:72]: lets make this from settings

File: src/navigation/Drawer.tsx [hunk 2]
```diff
@@ -32,6 +33,7 @@ export default function DrawerNavigation() {
             <Drawer.Screen name="settings" component={TestScreen} />
             <Drawer.Screen name="help" component={TestScreen} options={{ title: 'Help and Info' }} />
             <Drawer.Screen name="logout" component={TestScreen} />
+            <Drawer.Screen name="sso" options={{ title: 'test sso' }} component={SSOLoginScreen} />
```
Comment (theblockstalk) [inline src/navigation/Drawer.tsx:36]: Can you make the title be the app name `settings.config.appName`

Comment (Rebal67) [reply src/navigation/Drawer.tsx:36]: the tiitle won't be shown anyway header  is disabled

File: src/containers/SSOLoginContainer.tsx (hunk 1 above)
Comment (Rebal67) [reply src/containers/SSOLoginContainer.tsx]: changed

**Pull request merged**